src/main/java/frc/robot/subsystems/SwerveDrivetrain.java

```diff
@@ -105,6 +138,14 @@ public SwerveDrivetrain(SwerveIdConfig frontLeftConfig, SwerveIdConfig frontRigh
         Robot.getDiagnostics().addDiagnosable(new DiagAprilTags("AprilTags", "Apriltag Seen"));
         alignableTurnPid.enableContinuousInput(-180, 180);
         SmartDashboard.putBoolean("USE VISION",false);
+         NetworkTableInstance inst = NetworkTableInstance.getDefault();
```

Comment:
I think it may be better to move all the limelight readings into a different Vision subsystem.